config/routes.rb

```diff
@@ -14,9 +14,10 @@
       resources :classrooms, defaults: { format: :json } do
         resources :work_groups, shallow: true do
           resources :worksheets, except: :destroy, shallow: true
+          resources :messages, only: :create
```

Comment:
if we don't use the ID of the previous model, it doesn't make sense to nest it

Reply:
I was acting under the assumption I would use the work_group model for an association, I just wasn't sure where to yet 😅

Reply:
then if we have the work group model for association, then we need to include its id in the table :)
but see above - not sure if we really want to tie it to the work group or the specific worksheet